test/unit/services/test_usage_status.py

```diff
@@ -9,7 +9,10 @@
 import pytest
 from bson import ObjectId
 
-from inventory_management_system_api.models.usage_status import UsageStatusIn, UsageStatusOut
+from inventory_management_system_api.models.usage_status import (
+    UsageStatusIn,
+    UsageStatusOut,
+)
```

Comment:
Not sure why this happened black doesn't care but my vscode formats it back when I save?
```suggestion
from inventory_management_system_api.models.usage_status import UsageStatusIn, UsageStatusOut
```

Reply:
Could be the line length being the default instead of 120 on vscode's settings?